ssh/src/main/java/ch/cyberduck/core/sftp/openssh/OpenSSHAgentAuthenticator.java

```diff
@@ -55,7 +55,11 @@ public AgentProxy getProxy() {
     @Override
     public Collection<Identity> getIdentities() {
         if(null == proxy) {
-            log.warn(String.format("SSH agent %s is not running", this));
+            log.warn("Missing proxy reference");
+            return Collections.emptyList();
+        }
+        if(!proxy.isRunning()) {
```

Comment:
I think this additional conditional is breaking 1password for non IdentityFile uses. I don't think this check is needed for the original issue either.